python/bindings_so3.cpp

```diff
@@ -26,6 +26,15 @@ void wrap_SO3(py::module &m)
 
   SO3.def(py::init<const Scalar, const Scalar, const Scalar>());
   SO3.def(py::init<const Scalar, const Scalar, const Scalar, const Scalar>());
+  SO3.def(py::init([](const Eigen::Vector4d& quat) {
```

Comment:
```suggestion
  SO3.def(py::init([](const Eigen::Matrix<Scalar, 4, 1>& quat) {
```

Reply:
Done in https://github.com/artivis/manif/pull/211/commits/90e119e88c87fe624869b59e8d1e1e051c91349f